bootloader/src/pyi_utils.c

```diff
@@ -685,6 +685,8 @@ set_dynamic_library_path(const char* path)
     if (orig_path) {
         pyi_setenv(env_var_orig, orig_path);
         VS("LOADER: %s=%s\n", env_var_orig, orig_path);
+    } else {
+        VS("LOADER: Skipped %s\n", env_var_orig);
```

Comment:
This message should be more meaningful. Imagine looking at the debug messages and reading just "Skipped: LD_LIBRARY_PATH_ORIG" What does this mean? Something like "LD_LIBRARY_PATH was not set, thus not setting LD_LIBRARY_PATH_ORIG"  would make more sense for me.